wemake_python_styleguide/violations/best_practices.py

```diff
@@ -1941,3 +1943,30 @@ class FloatKeyViolation(ASTViolation):
 
     error_template = 'Found float used as a key'
     code = 449
+
+
+@final
+class UndescriptiveSliceOperationViolation(ASTViolation):
+    """
+    Forbids use of arcane slice operations with more descriptive alternatives.
+
+    Reasoning:
+        We do this to make code more readable.
+
+    Solution:
+        Replace arcane slice operations with descriptive list functions.
+
+    Example::
+
+        # Correct:
+        items.reverse()
+
+        # Wrong:
+        items[::-1]
+
+    .. versionadded:: 0.13.0
+
+    """
+
+    error_template = 'Found undescriptive slice operation'
+    code = 500
```

Comment:
Code is not correct. This violation should be located in `refactoring.py` and have a correct code.